helm/dashboards/dashboards/shared/private/loki-cost-estimation.json

```diff
@@ -42,45 +42,44 @@
   "liveNow": false,
   "panels": [
     {
-      "collapsed": true,
+      "collapsed": false,
```

Comment:
Some panel was collapsed and is not anymore. Is it expected?

Reply:
Not at all ! Let me change this